### PR TITLE
Add #cdn support to demo images

### PR DESCRIPTION
**Pull request by sampoder**
I have no clue, this is untested, but this might add cdn support (smth I was chatting about with @zachlatta)

what this does is allows you to give a url in your yaml front matter(? is that what its called) and then it will show

pls dont break thanks js

### CODE REVIEW

File: components/listing.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const WorkshopCard = ({ slug, name, description, section }) => (
       {(section === 'starters' || section === 'web') && (
         <Image
           alt="Demo"
-          src={`/content/workshops/${slug}/img/demo.png`}
+          src={(img ? img : `/content/workshops/${slug}/img/demo.png`)}
```
Comment (lachlanjc) [inline components/listing.js:27]:
```suggestion
          src={img || `/content/workshops/${slug}/img/demo.png`}
```

File: lib/data.js [hunk 2]
```diff
@@ -44,8 +44,13 @@ export const getWorkshopSections = () =>
...
       let md = getWorkshopFile(slug)
-      let { name, description } = matter(md).data
-      workshops.push({ slug, name, description })
+      let { name, description, img } = matter(md).data
+      if (typeof img === 'undefined') {
+        workshops.push({ slug, name, description })
+      }
+      else{
+        workshops.push({ slug, name, description, img })
+      }
```
Comment (lachlanjc) [inline lib/data.js:53]:
```suggestion
      workshops.push({ slug, name, description, img })
```

**Review by lachlanjc — APPROVED** [review]
Thanks a bunch!

**Comment by sampoder** [comment]
epic! no worries!

**Pull request merged**